### DISCUSSION TITLE
Slack Integration?

### DISCUSSION THREAD

**Discussion by justin808** [shakacode/shakapacker#180] [Announcements]
How can we enable these messages to go to [React Plus Rails Slack](https://join.slack.com/t/reactrails/shared_invite/enQtNjY3NTczMjczNzYxLTlmYjdiZmY3MTVlMzU2YWE0OWM0MzNiZDI0MzdkZGFiZTFkYTFkOGVjODBmOWEyYWQ3MzA2NGE1YWJjNmVlMGE)?

When you create a discussion, please post a link on our Slack!

I failed to notice the messages here! 😱

**Comment by Judahmeek** [comment]
https://github.com/orgs/community/discussions/2844#discussioncomment-2061814
